packages/terra-site/src/examples/dialog/DialogDefault.jsx

```diff
@@ -1,17 +1,23 @@
+/* eslint-disable no-alert */
 import React from 'react';
 import Dialog from 'terra-dialog';
 
-const header = <span>Header Stuff</span>;
+const header = 'Header Stuff';
 
-const footer = <span>Footer Stuff</span>;
+const footer = <Dialog.Footer>Footer Stuff</Dialog.Footer>;
+
+const body = (<Dialog.Body>
+  <p>This is my body content.</p>
+  <p>This is some more content.</p>
+</Dialog.Body>);
 
 const DialogDefault = () => (
   <div style={{ height: '200px', width: '350px', border: 'dashed' }}>
```

Comment:
Given how large this hash is, I think it's best to extract it to a CSS file. Also, should the border be themeable? @neilpfeiffer

Reply:
Disregard. This is an example page.